pandas/core/indexes/multi.py

```diff
@@ -68,6 +68,32 @@ class MultiIndex(Index):
         Copy the meta-data
     verify_integrity : boolean, default True
         Check that the levels/labels are consistent and valid
+
+    Examples
+    ---------
+    A new ``MultiIndex`` is typically constructed using one of the helper
+    methods :attr:`MultiIndex.from_arrays``, :attr:`MultiIndex.from_product``
+    and :attr:`MultiIndex.from_tuples``. For example (using ``.from_arrays``):
+
+    >>> arrays = [[1, 1, 2, 2], ['red', 'blue', 'red', 'blue']]
+    >>> MultiIndex.from_arrays(arrays, names=('number', 'color'))
+    MultiIndex(levels=[[1, 2], ['blue', 'red']],
+           labels=[[0, 0, 1, 1], [1, 0, 1, 0]],
+           names=['number', 'color'])
+
+    See further examples for how to construct a MultiIndex in the doc strings
+    of the mentioned helper methods.
+
+    Notes
+    -----
+    See the :ref:`user guide <advanced>` for more.
```

Comment:
I think you have to put the full http link here (technically you don't for the built docs), but in a console this won't work.

Reply:
Ok. done.